lib/randutils.c

```diff
@@ -74,6 +74,10 @@ static void crank_random(void)
 		rand();
 }
 
+#if !defined(O_CLOEXEC)
+#define O_CLOEXEC 0
+#endif
+
```

Comment:
We have this falback in c.h and this include file is included to randutils.c. So, why we need to duplicate this ifdef here?